### PR TITLE
pr triage: workflow documentation

### PR DESCRIPTION
**Pull request by kpsherva**

### CODE REVIEW

**Comment by fenekku** [comment]
Coordination: any other member is welcome to review (I just didn't know / couldn't assign the github handle of everyone)

File: docs/maintenance/pr-community-board.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- **Maintainer**: Person who has merging rights in the InvenioRDM repositories, responsible for integrating the pull requests into the product.
+- **Release manager**: Person in charge of the current main InvenioAppRDM release.
+
+## Columns
+
+- **Triage**: **Submitters** place the PRs in this column. PR is ready to be prioritised by the **coordinator**. 
+- **Work in progress**: Changes/suggestions were requested by the reviewers, the submitter is addressing them.
+- **To review**: PR is ready to be reviewed by the reviewers.
+- **Discussion**:  PR needs to be re-discussed in terms of solutions - fe. there are different opinions on the implementation.
+- **Approved**: PR was approved, two reviewers agree on integrating it. Ready to be merged.
```
Comment (tmorrell) [inline docs/maintenance/pr-community-board.md:30]: We discussed some cases where one reviewer is sufficient. Not sure how to include that here without being too wordy.

File: docs/maintenance/pr-community-board.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- **To release**: PR is ready to be released. **Maintainers** or **release manager** decide on when it should happen. PRs might be merged or not merged yet.
+- **Released/done**: Archive of recently integrated PRs - useful for tracking the latest changes
```
Comment (tmorrell) [inline docs/maintenance/pr-community-board.md:32]: One question that came up with @fenekku that we haven't discussed is how long PR's will stay in the done column. What does "latest" mean in terms of time?

File: docs/maintenance/pr-community-board.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- **Release manager**: Person in charge of the current main InvenioAppRDM release.
+
+## Columns
+
+- **Triage**: **Submitters** place the PRs in this column. PR is ready to be prioritised by the **coordinator**. 
+- **Work in progress**: Changes/suggestions were requested by the reviewers, the submitter is addressing them.
+- **To review**: PR is ready to be reviewed by the reviewers.
+- **Discussion**:  PR needs to be re-discussed in terms of solutions - fe. there are different opinions on the implementation.
+- **Approved**: PR was approved, two reviewers agree on integrating it. Ready to be merged.
+- **To release**: PR is ready to be released. **Maintainers** or **release manager** decide on when it should happen. PRs might be merged or not merged yet.
```
Comment (tmorrell) [inline docs/maintenance/pr-community-board.md:31]: There is a bit of a discrepancy between this line and the figure. Should approved but non-merged blocked PRs go in "Approved" or "To release"?

**Review by tmorrell — APPROVED** [review]
Great summary! Just a couple of questions on details

File: mkdocs.yml [hunk 4]
```diff
@@ -175,6 +175,7 @@ nav:
       - Overview: maintenance/index.md
       - Newcomers: maintenance/newcomers.md
       - Sprintboard workflow: maintenance/board-workflow.md
+      - PR Community board: maintenance/pr-community-board.md
```
Comment (fenekku) [inline mkdocs.yml:178]:
```suggestion
      - Community PR board: maintenance/pr-community-board.md
```

**Review by fenekku — APPROVED** [review]
This is great. Most of my comments are just wording improvements. We can talk some more about things next week too.

I also forgot to mention, that I ended up providing a list of reviewers/maintainers per repo on the board page so you don't have to / you can just edit it.

Thanks!

File: docs/maintenance/pr-community-board.md (hunk 1 above)
Comment (kpsherva) [reply docs/maintenance/pr-community-board.md:30]: I could add a longer description below, but this is potentially something that coordinator or maintainer could decide per case.

File: docs/maintenance/pr-community-board.md (hunk 2 above)
Comment (kpsherva) [reply docs/maintenance/pr-community-board.md:32]: the one since the last major invenio-app-rdm release. Release manager could be the one responsible to archive the column when the major is released

File: docs/maintenance/index.md [hunk 5]
```diff
@@ -8,6 +8,7 @@ This guide is intended for maintainers and developers of InvenioRDM itself.
 
 - **[Newcomers](newcomers.md)**: A quick overview for new core developers.
 - **[Sprintboard workflow](board-workflow.md)**: A reference for our sprintboard workflow.
+- **[PR Community board](pr-community-board.md)**: A reference for our PR community workflow.
```
[the remark this answers is not in the document]
Comment (kpsherva) [reply docs/maintenance/index.md:11]: Great opportunity for me to practice grammar with help of the native speaker, so thank you a lot! :)

File: docs/maintenance/pr-community-board.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+This guide is intended for contributors, coordinators and developers of InvenioRDM.
+
+**Scope**
+
+The guide covers the recommended way of coordinating community pull requests integration within the InvenioRDM. The goal of the project is to respond to the commmunity need of facilitating the review and integration processes of the **pull requests which are not the part of the main sprint**.
+
+## Board
+
+You'll find the latest board [here](https://github.com/orgs/inveniosoftware/projects/109/views/1).
```
[the remark this answers is not in the document]
Comment (kpsherva) [reply docs/maintenance/pr-community-board.md:13]: It would be helpful for the release manager to keep the PRs in the column until major release of invenio-app-rdm. It is very helpful to create release notes.

File: docs/maintenance/pr-community-board.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+- **Reviewer**: Person who contributes by reviewing contributors' work.  
+- **Maintainer**: Person who has merging rights in the InvenioRDM repositories, responsible for integrating the pull requests into the product.
+- **Release manager**: Person in charge of the current main invenio-app-rdm release.
+
+## Columns
+
+- **Triage**: **Submitters** place the PRs in this column. PR is ready to be prioritised by the **coordinator**. 
+- **Work in progress**: Changes/suggestions were requested by the reviewers, the submitter is addressing them.
+- **To review**: PR is ready to be reviewed by the reviewers.
+- **Discussion**:  PR needs to be re-discussed in terms of solutions - e.g., there are different opinions on the implementation.
```
Comment (chokribr) [inline docs/maintenance/pr-community-board.md:29]: If discussion are not relevants PR may be assigned a specific  status
